cpp/src/strings/convert/utilities.cuh

```diff
@@ -96,8 +96,9 @@ __device__ inline size_type integer_to_string(IntegerType value, char* d_buffer)
 template <typename IntegerType>
 constexpr size_type count_digits(IntegerType value)
 {
+  // TODO definitely broken
```

Comment:
Done-ish. I figure because it was already in a `constexpr` function it doesn't need to be isolated, but maybe my reasoning is off her. Let me know what you think.